assets/primitives/asset_user.json

```diff
@@ -0,0 +1,604 @@
+      "e2": 34.640751626287
+    },
+    {
+      "id": 481,
+      "n1": 145,
+      "n2": 146,
+      "type": "bottle_link",
+      "bottle_type": "Pipe(60cm)",
+      "piston_group": -1,
+      "e1": 34.640751754777895,
+      "e2": 34.640751754777895
+    },
+    {
+      "id": 482,
+      "n1": 146,
+      "n2": 147,
+      "type": "bottle_link",
+      "bottle_type": "Pipe(60cm)",
+      "piston_group": -1,
+      "e1": 34.6407517547778,
+      "e2": 34.6407517547778
+    },
+    {
+      "id": 483,
+      "n1": 147,
+      "n2": 145,
+      "type": "bottle_link",
+      "bottle_type": "Pipe(60cm)",
+      "piston_group": -1,
+      "e1": 34.64075175477794,
+      "e2": 34.64075175477794
+    },
+    {
+      "id": 484,
+      "n1": 146,
+      "n2": 135,
+      "type": "bottle_link",
+      "bottle_type": "Pipe(60cm)",
+      "piston_group": -1,
+      "e1": 34.64075178047614,
+      "e2": 34.64075178047614
+    },
+    {
+      "id": 485,
+      "n1": 147,
+      "n2": 135,
+      "type": "bottle_link",
+      "bottle_type": "Pipe(60cm)",
+      "piston_group": -1,
+      "e1": 34.64075208885391,
+      "e2": 34.64075208885391
+    },
+    {
+      "id": 486,
+      "n1": 145,
+      "n2": 141,
+      "type": "bottle_link",
+      "bottle_type": "Pipe(60cm)",
+      "piston_group": -1,
+      "e1": 34.64075070115354,
+      "e2": 34.64075070115354
+    },
+    {
+      "id": 487,
+      "n1": 147,
+      "n2": 141,
+      "type": "bottle_link",
+      "bottle_type": "Pipe(60cm)",
+      "piston_group": -1,
+      "e1": 34.64075239723218,
+      "e2": 34.64075239723218
+    },
+    {
+      "id": 488,
+      "n1": 146,
+      "n2": 140,
+      "type": "bottle_link",
+      "bottle_type": "Pipe(60cm)",
+      "piston_group": -1,
+      "e1": 34.640751626287184,
+      "e2": 34.640751626287184
+    },
+    {
+      "id": 489,
+      "n1": 145,
+      "n2": 140,
+      "type": "bottle_link",
+      "bottle_type": "Pipe(60cm)",
+      "piston_group": -1,
+      "e1": 34.64075162628732,
+      "e2": 34.64075162628732
+    },
+    {
+      "id": 490,
+      "n1": 139,
+      "n2": 146,
+      "type": "bottle_link",
+      "bottle_type": "Pipe(60cm)",
+      "piston_group": -1,
+      "e1": 34.64075062405891,
+      "e2": 34.64075062405891
+    },
+    {
+      "id": 491,
+      "n1": 138,
+      "n2": 142,
+      "type": "bottle_link",
+      "bottle_type": "Pipe(60cm)",
+      "piston_group": -1,
+      "e1": 34.64075062405873,
+      "e2": 34.64075062405873
+    },
+    {
+      "id": 492,
+      "n1": 145,
+      "n2": 143,
+      "type": "bottle_link",
+      "bottle_type": "Pipe(60cm)",
+      "piston_group": -1,
+      "e1": 34.640753502253055,
+      "e2": 34.640753502253055
+    }
+  ],
+  "animation": "[[0,[{\"time\":0,\"value\":0.5},{\"time\":8,\"value\":0.5}]]]",
+  "standard_surface": [
+    145,
+    143,
+    141
+  ],
+  "spring_constants": {
+  },
+  "mounted_users": [
+    {
+      "id": 140,
+      "name": "boy",
```

Comment:
you use the name to load the `skp` file, right? If so, please consider using the whole filename in here...

Reply:
See below, agreed